qa/tasks/vstart_runner.py

```diff
@@ -955,7 +955,7 @@ def flush(self):
     # For the benefit of polling tests like test_full -- in teuthology land we set this
     # in a .yaml, here it's just a hardcoded thing for the developer's pleasure.
     remote.run(args=[os.path.join(BIN_PREFIX, "ceph"), "tell", "osd.*", "injectargs", "--osd-mon-report-interval-max", "5"])
-    ceph_cluster.set_ceph_conf("osd", "osd_mon_report_interval_max", "5")
```

Comment:
You missed the "--osd-mon-report-interval-max" right above this line (although I don't think it has much effect).

Otherwise it looks good.